db/migrate/20250701092007_create_case_record.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class CreateCaseRecord < ActiveRecord::Migration[7.2]
+  def change
+    create_table :case_records, id: false do |t|
+      t.uuid :id, primary_key: true
+      t.references :local_authority, null: false
+      t.string :caseable_type, null: false
+      t.bigint :caseable_id, null: false
```

Comment:
I think we want an index on these two columns so doing `enforcement.case_record` doesn't result in a table scan